### ISSUE TITLE
FR: Custom claims support

### ISSUE THREAD

**Issue by Mistic92** [firebase/firebase-admin-java#80]
According to docs, for now setting custom claims is available only for JS. 
Would be nice to have this option also in Java

https://firebase.google.com/docs/auth/admin/custom-claims

**Comment by hiranya911** [comment]
This is being developed in the hkj-custom-claims branch. But it will be a while before we can get it released. There are couple of other things scheduled to release before that. But, rest assured that it is coming :)

**Comment by sowdri** [comment]
Is there any workaround which this is getting developed? I can set the custom claims in the console, just need the ability to read those claims on the server.

**Comment by hiranya911** [comment]
@sowdri I don't see one, short of using the REST API directly. The relevant PR #92 is currently under review though. I can try to get its review expedited, and released in a couple of weeks. If you do decide to use the REST API in the meantime, check the `FirebaseUserManager` class, and see how it makes REST calls to the ID toolkit API.

**Comment by hiranya911** [comment]
This was released today: https://firebase.google.com/support/release-notes/admin/java#5.6.0
